src/SimpleExecutionController.cpp

```diff
@@ -239,27 +244,60 @@ void SimpleExecutionController::processEventCompoundJobCompletion(std::shared_pt
     std::string execution_host = (*(event->job->getActions().begin()))->getExecutionHistory().top().physical_execution_host;
 
     /* Remove all actions from memory and compute incremental output values in one loop */
-    double incr_compute_time = 0.;
+    double incr_compute_time = -9999.;
     double incr_infile_transfertime = 0.;
     double incr_infile_size = 0.;
     double incr_outfile_transfertime = 0.;
     double incr_outfile_size = 0.;
     double start_date = DBL_MAX;
-    double end_date = 0;
+    double end_date = DBL_MIN;
+    double hitrate = -9999.;
+
+    bool found_computation_action = false;
```

Comment:
Maybe it would be good to introduce some standard default values for the complete packages somewhere in Utils.h or so

Reply:
This has been implemented in commit [a61f29f](https://github.com/HEPCompSim/DistCacheSim/pull/29/commits/a61f29fc43032f23f121009daa64e40ff91df4b2).